src/Sequence.cc

```diff
@@ -748,6 +748,14 @@ void Sequence::set_step_setup_script(gul14::string_view step_setup_script)
     step_setup_script_.assign(step_setup_script.data(), step_setup_script.size());
 }
 
+void Sequence::set_tags(const std::vector<Tag>& new_tags)
+{
+    auto tags = new_tags;
```

Comment:
Is there a reason why this is done in a dedicated temporary instead of `tags_ = new_tags; ...`?
Maybe exception safety? :thinking:

Reply:
Well, yes, that was my original idea. But on the other hand: The only exception that could be thrown is probably `std::bad_alloc`. So maybe it is too much complication for nothing. I can change it it you like, otherwise I would just leave it as it is.